lms/djangoapps/grades/tests/test_new.py

```diff
@@ -205,8 +205,8 @@ def test_save_and_load(self):
         input_grade.init_from_structure(
             self.request.user,
             self.course_structure,
-            self.subsection_grade_factory._scores_client,  # pylint: disable=protected-access
             self.subsection_grade_factory._submissions_scores,  # pylint: disable=protected-access
+            self.subsection_grade_factory._csm_scores,  # pylint: disable=protected-access
```

Comment:
Nit: In test files, I tend to prefer disabling the protected-access lint at the top of the file.